fedimint-api/src/core.rs

```diff
@@ -36,6 +36,8 @@ pub const MODULE_KEY_MINT: u16 = 1;
 pub const MODULE_KEY_LN: u16 = 2;
 // not really a module
 pub const MODULE_KEY_GLOBAL: u16 = 1024;
+// need a key for the client
+pub const CLIENT_KEY: u16 = 3;
```

Comment:
Oh. I see. I thought it might be some db key ... but actually they are one of the same... Nvmd.